tests/integration/fixtures/mutation-insert.graphql

```diff
@@ -6,27 +6,33 @@ mutation Insert {
 
 fragment payloadThing on InsertThingPayload {
   clientMutationId
-  thing {
-    id
-    rowId
-    note
+  thingEdge(orderBy: ROW_ID) {
+    cursor
+    node {
+      id
+      rowId
+      note
+    }
   }
 }
 
 fragment payloadRelation on InsertRelationPayload {
   clientMutationId
-  relation {
-    aThingId
-    bThingId
```

Comment:
Ditto, generally removing tests is a bad sign because it means we might break things for some users.